src/options.js

```diff
@@ -0,0 +1,15 @@
+var path = require('path');
+var pkg = require('../package.json');
+
+module.exports = {
+  // cmd, homepage, bugs all pulled from package.json
+  cmd: 'sagastandard',
+  version: pkg.version,
+  homepage: pkg.homepage,
+  bugs: pkg.bugs.url,
+  tagline: 'Semicolons For All!',
+  eslint: require('eslint'),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Use a standard "import" statement instead of "require(...)". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=javascript%3AS3533)